File: StackAndQueus/stack.cs
using System;

namespace Solution
{
    class Program
    {
        static int SIZE = 6;
        static int[] S = new int[SIZE];
        static int top = -1;
        static void Main(string[] args)
        {
         
            while(true){
                
                int n = int.Parse(Console.ReadLine());

                switch(n) {
                    case 1:
                        PUSH(S, n);
                        break;
                    case 3:
                        PUSH(S, n);
                        break;
                    case 4:
                        PUSH(S, n);
                        break;
                    case 8:
                        PUSH(S, n);
                        break;
                    case 5:
                        POP(S);
                        break;
                    default:
                        EMPTY(S);
                        break;
                }
            }
        }
        static void PUSH(int[] S, int x) {
            top++;
            S[top] = x;
            Console.WriteLine("Add :" + S[top]);
        }
        static void POP(int[] S) {
            if(top == -1) {
                Console.WriteLine("Stack is underflow");
                return;
            }
            else {
                top--;
                Console.WriteLine("Delete: " + top);
            }
        }
        static bool EMPTY(int[] S) {
            Console.WriteLine("Stack is epmty!");
            return S.Length == 0 ? true : false;
        }
    }
}

File: StackAndQueus/stack_.c

#include <stdio.h>
#include <stdlib.h>



int size;
void push ( int x);
void pop();
int isempty();
int *a = NULL;


int main() {
    
    int n, r;
    int add;
    a = malloc(1 * sizeof(int));
    
    while(1) {
        printf("Select  \
                    1) add \
                    2) delete stack:");
        printf("\n n = ");
            scanf("%d", &n);
            printf("\n");
        switch (n)
        {
        case 1:
        
         printf("a = ");
         scanf("%d", &add);
            push(add);
            break;
        case 2:
            pop();
            break;
        default:
         printf("try again!");
            break;
        }
        printf("\n");
    }


}
void push(int x) {
    a[++size] = x;
    printf("add: %d" , a[size]);
    if(a == NULL) 
        isempty();
}
void pop() {
    if(size == -1) {
        printf("stack is underflow\n");
        if(isempty() == 0)
            return;
    }
    else { 
        printf("deleted: %d ", a[size]);
        --size; 
    }
}
int isempty() {
    printf("stack is empty!");
    return size == 0 ? 1 : 0;
}